deel-3-webpagina/index.html

```diff
@@ -7,10 +7,33 @@
     <title>Flexbox opdracht 3</title>
 </head>
 <body>
-    <div>
-        <h1>All Your Business Data in One Place</h1>
-        <h3>Sync and store all your business data. Explore your data with SQL or visualize it with your favorite BI or
-            analytics tools.</h3>
-    </div>
+
+<nav>
+    <a class="analytica"> Analytica</a>
+    <a class="navbalk"> About Us</a>
+    <a class="navbalk"> Strategy</a>
+    <a class="navbalk"> Prices</a>
+    <a class="navbalk"> Contact</a>
+</nav>
+
+<div class="titels">
```

Comment:
Alle inhoud had in een section geplaatst kunnen worden en dan hadden de articles in een container div geplaats kunnen worden wat zorgt voor meer dynamica.